README.md

```diff
@@ -1,35 +1,34 @@
-# Full-Screen Fútball Fan
+# Full-Screen Fútbol Fan
 
 ## Description
-This project is a work in progress football stats app focused on MLS (Major League Soccer). It is intended for use by football fans to look up team stats and gather information about upcoming matches.
+This project is a football stats app focused on MLS (Major League Soccer). It is intended for use by football fans to look up team stats and gather information about upcoming matches.
 
-### Features
-
-### Planned Features
+## Features
 * Homepage
-    * If a team is favorited, team stats and upcoming matches are displayed
-* Stats page
-    * Search for teams
-    * Favorited team displays on stats on pageload
+    * The next upcoming league match is displayed.
+    * If a team is favorited, it's next match is displayed. The league match can then be displayed by using the available button.
```

Comment:
"The available button" might be confusing to someone looking at this for the first time.